Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: deps install all
+
+ifndef NASHPATH
+	$(error $$NASHPATH is not set)
+endif
+
+ifndef GOPATH
+	$(error $$GOPATH is not set)
+endif
+
+all: deps install
+
+deps:
+	go get github.com/c0defellas/enzo/cmd/...
+
+install:
```

Comment:
I found this install rule odd. I agree with @katcipis that this could leave the install in an inconsistent state... or be harmful in case of overwritten user's files.

I think the only required rule in the Makefile is "deps" for now.